src/main/java/duke/Main.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import duke.ui.Ui;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+import static duke.common.Messages.filePath;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+    static Boolean isScreenLoaded = false;
+//    private MainWindow mainWindow = new MainWindow();
+//    private Ui ui = new Ui(mainWindow);
+//    private Duke duke = new Duke(ui);
+
+    @Override
+    public void start(Stage stage) throws Exception {
+        FXMLLoader fxmlLoader = new FXMLLoader(Main.class.getResource("/view/MainWindow.fxml"));
```

Comment:
You are violating SLAP here. If the responsibility of this method is to call other objects' method to start the application, then it should not deal with creation of objects to follow SLAP.